wdio/config/custom-commands.js

```diff
@@ -71,6 +71,18 @@ module.exports = function (browser, capabilities, specs) {
     }
   });
 
+  browser.addCommand('checkSelectedLanguage', (language) => {
+    browser.waitForVisible('.languages');
+    const activeLanguage= browser.getText('.btn-group.languages .btn.btn-default.active');
+    assert.equal(language, activeLanguage);
+  });
+
+  browser.addCommand('checkSelectedUserType', (userType) => {
```

Comment:
I don't think you need a custom command. You can do something like [this](https://github.com/Gizra/circuit-bid/blob/47a77f4b893c3024ca11bc1025994e68432834cf/client/test/specs/mail-auction.js#L5-L10)